rocketpy/mathutils/function.py

```diff
@@ -3238,14 +3327,17 @@ def __validate_interpolation(self, interpolation):
                 interpolation = "spline"
         ## multiple dimensions
         elif self.__dom_dim__ > 1:
-            if interpolation not in [None, "shepard"]:
+            if interpolation is None:
+                interpolation = "shepard"
+            if interpolation not in ["shepard", "linear", "rbf"]:
                 warnings.warn(
                     (
-                        "Interpolation method set to 'shepard'. Only 'shepard' "
-                        "interpolation is supported for multiple dimensions."
+                        "Interpolation method set to 'shepard'. The methods "
+                        "'linear' and 'shepard' are supported for multiple "
+                        "dimensions."
                     ),
                 )
-            interpolation = "shepard"
+                interpolation = "shepard"
```

Comment:
regarding the warning message, shouldn't we also include the rbf option as an available option?

Reply:
We should! Solved in [86458aa](https://github.com/RocketPy-Team/RocketPy/pull/691/commits/86458aab47331eaeb59f9c61795f84548c5eb90c)